### DISCUSSION TITLE
Upgrading pl 0.8.5 code to current version

### DISCUSSION THREAD

**Discussion by SapirWeissbuch** [Lightning-AI/pytorch-lightning#7720] [General]
Hi,

Is there a simple way or tool to upgrade old pytorch-lightning code (version 0.8.5, to be specific) to the current pytorch lightning version?
A guide or tips could also be of help.

Thanks!

**Comment by justusschock** [comment]
Hi @SapirWeissbuch 

I'm afraid there isn't. Version 0.8.5 is already quite old. I think the easiest thing is to look through all the (major) releases since then and to make a list of breaking changes that could affect you. Once you reached 1.0.x the changes on the API side should become less severe but since you're not yet on a stable version this is probably the best option you got.
